doc/api/http.md

```diff
@@ -107,6 +107,13 @@ http.get({
 });
 ```
 
+<!-- YAML
+added: v10.14.0
```

Comment:
wasn't sure what version to specify here since this is added in multiple releases (backports to 6) as well as 8, 10 and 11 (specifically v6.15.0, v8.14.0, v10.14.0, v11.3.0)

Reply:
@nodejs/documentation How do we usually handle that?